test/bigtest/tests/settings-test.js

```diff
@@ -0,0 +1,21 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import SettingsInteractor from '../interactors/settings';
+
+describe('Settings', () => {
+  const settings = new SettingsInteractor();
+
+  setupApplication();
+
+  describe('general', () => {
+    beforeEach(function () {
+      this.visit('/settings/oaipmh/general');
+    });
+
+    it('has a general settings message', () => {
```

Comment:
It is a good idea to start the it block with the word `should`.

```suggestion
    it('should have a general settings message', () => {
```

Reply:
fixed